Solutions/OracleWebLogicServer/Parsers/OracleWebLogicServerEvent.txt

```diff
@@ -41,8 +41,8 @@ OracleWebLogicServer_CL
 | extend HttpRequestMethod = tostring(SubEventData1[0])
 | extend UrlOriginal = tostring(SubEventData1[1])
 | extend HttpVersion = tostring(SubEventData1[2])
-| extend HttpStatusCode = tostring(SubEventData2[0])
-| extend HttpResponseBodyBytes = tostring(SubEventData2[1])
+| extend HttpStatusCode = toint(SubEventData2[0])
```

Comment:
Can you please provide the sample data for this raw data field where we can validate the parser and get the results for all the columns?

Reply:
Added raw sample data under Sample Data/Custom/OracleWebLogicServer folder